lib/soapy_cake/time_converter.rb

```diff
@@ -0,0 +1,35 @@
+module SoapyCake
+  class TimeConverter
+    def initialize(time_zone, time_offset = nil)
+      if time_offset
+        self.class.print_deprecation_warning
+
+        # Etc/GMT time zones have their sign reversed
+        time_zone = format('Etc/GMT%+d', -time_offset.to_i)
+      end
+
+      fail Error, 'Cake time zone missing' if time_zone.blank?
+      @zone = ActiveSupport::TimeZone[time_zone]
+    end
+
+    def to_cake(date)
```

Comment:
`TimeConverter` should not be used outside of soapy_cake. I just extracted the logic from `Request` and `Response` to this class. The new contract is really simple:
- Define `CAKE_TIME_ZONE`
- Specify dates with any timezone in requests (will be converted to the correct CAKE timezone)
- Responses contain UTC dates which can be converted however you like

Why would I need this class outside of soapy_cake?